dkms.8.in

```diff
@@ -651,6 +651,16 @@ This optional directive functions very similarly to
 except that it matches against the kernel architecture. For example, if you set
 it to ="i.86", your module would not be built for ia32e, x86_64, amd64, s390, etc.
 .TP
+.B BUILD_EXCLUSIVE_CONFIG=
+This optional directive allows you to specify a space separated list of
+kernel configuration options ("CONFIG_FOO") that must be enabled in the
+targeted kernel (either compiled in or built as a module) or absent
+(if prefixed with an exclamation mark, e.g. "!CONFIG_BAR") in order to
+build the module.
+For example, if you set it as ="CONFIG_PCI !CONFIG_PREEMPT_RT", your module
+would only be built for kernels that have PCI enabled, but the RT patchset
+disabled.
+.TP
```

Comment:
Nit: add mention that this requires the `.config` file to be around. This will be OK with Arch, but I have not checked other distributions - or maybe people doing custom kernels should be aware.

Reply:
I've now mentioned the `.config` file, but I doubt that you can build a (usable) module without a `.config` available.